Makefile

```diff
@@ -1,6 +1,6 @@
 data/whitelisted_ext_idefs.json: 
 	python3 wikidatarefisland/run.py --step ss1 --output "whitelisted_ext_idefs.json"
-data/pipe1.json:
-    python3 wikidatarefisland/run.py --step itemextract --input "dump.json" --output "pipe1.jsonl"
+data/extracted_unreferenced_statements.jsonl:
```

Comment:
as your script depends on the result of ss1, I think you should add that step as a dependency

Reply:
Gotcha, so you think no need to run it if missing? Makes sense to me